### DISCUSSION TITLE
How to downgrade/restore Ant Media server to previous version after upgrading?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5354] [FAQs] [answered]
I recently upgraded my Ant Media Server instance from v2.5.3 to v2.6.1. I was wondering if there is a way to restore and go back to v2.5.3 again.

**Comment by yashtandon113** [comment, marked as answer]
Thanks @Mohit-3196 for asking this question.

When you make a fresh installation or upgrade over an older version, the previous installation backed up in the **/usr/local** directory with a timestamp value like antmedia-backup-2022-11-18_15-42-54. Now, in order to restore the previous installation, kindly follow the below commands:

```
sudo systemctl stop antmedia
sudo rm -rf /usr/local/antmedia
sudo cp -p -R /usr/local/antmedia-backup_folder/ /usr/local/antmedia
sudo chown -R antmedia:antmedia /usr/local/antmedia/
sudo systemctl start antmedia
```
